cypress/support/testData.js

```diff
@@ -0,0 +1,4 @@
+export const user = {
+  username: 'user675',
+  password: 'qwertY123!'
+};
\ No newline at end of file
```

Comment:
It is recommended to have a newline at the end of the file for better compatibility across different systems and editors. 😊